admin/client/src/components/Accordion/AccordionGroup.js

```diff
@@ -15,12 +15,10 @@ class AccordionGroup extends SilverStripeComponent {
       'aria-labelledby': headerID,
     };
     return (
-      <div className="accordion-group">
-        <h6 className="accordion-group__title" role="tab" id={headerID}>
-          <a data-toggle="collapse" href={href} aria-expanded="true" aria-controls={listID}>
-            {this.props.title}
-          </a>
-        </h6>
+      <div className="accordion">
```

Comment:
Hmm, I think I assumed the component was called Accordion as there is also an Accordion.js file which wraps AccordionGroup.